### ISSUE TITLE
Error: You attempted to overwrite the built-in helper “array”.

### ISSUE THREAD

**Issue by rafael-mindreau** [adopted-ember-addons/ember-light-table#731]
Using Ember `3.17.1` with ELT `2.0.0-beta.5` we get the following error:

```
Uncaught Error: Assertion Failed: You attempted to overwrite the built-in helper "array" which is not allowed. Please rename the helper.
```

Tracking this down via [this issue](https://stackoverflow.com/questions/60746799/error-you-attempted-to-overwrite-the-built-in-helper-array-where-is-this-com), it originates from using `ember-composable-helpers` on a lower version than `3.x.x`. When I manually force this in the package-lock to be updated, everything works again.

Can you check if ELT runs perfect with the latest version of `ember-composable-helpers`?

**Comment by fpauser** [comment]
ELT seems to work with the latest `ember-composable-helpers`.
As a workaround I added this to the package.json to force yarn to use the latest version of `ember-composable-helpers`:
```
  "resolutions": {
    "ember-composable-helpers": "^3.1.1"
  }
```

**Comment by MikiDi** [comment]
see #732

**Comment by fran-worley** [comment]
The best solution to this is to replace our one use of the compute helper and drop the composable helpers dependency. The new version of the addon requires ember 3.13 which limits our ability to support older versions of Ember. 

I'll try and get this fixed for the new v2 release.

**Comment by rwwagner90** [comment]
Any updates on this? Anything I can do to help?

**Comment by hefox** [comment]
confirming issue & workaround

**Comment by rwwagner90** [comment]
@fran-worley I see you merged a PR. Does that resolve this?

**Comment by richard-viney** [comment]
AFAIK this is resolved but only on the `2-x` branch, not in an actual 2.x beta that has been tagged/released.

**Comment by richard-viney** [comment]
It's also fixed in the `3-x` branch and is included in the `3.0.0-beta-0` release that's available on NPM.

**Comment by rwwagner90** [comment]
We have both a 2 beta and a 3 beta? Interesting.

**Comment by fran-worley** [comment]
@rwwagner90 I will release final v2 when I have time to get a stable build on older versions of ember. v3 beta supports 3.12+ (at present) and will soon have breaking API changes to formally support angle bracket syntax. 

My development time is really tight at the moment so if anyone wants to lend a hand then we could turn things around a bit quicker! 😸

**Comment by rwwagner90** [comment]
@fran-worley I propose we don't worry about v2, and we merge all the latest and greatest stuff from 3-x into master. It'll make maintaining this much easier and it's typical to have the newest code in master, so people will not get confused looking for the new stuff.

**Comment by maxwondercorn** [comment]
This issue was resolved in the branches merged in pr #775 and is  included in the unpublished `3.0.0 Beta.1 prerelease`.

**Comment by rwwagner90** [comment]
@maxwondercorn 3.0.0-beta.1 is published, just under beta.
